spec/vmfloaty/utils_spec.rb

```diff
@@ -31,4 +31,61 @@
       expect(Utils.generate_os_hash(host_arg)).to be_empty
     end
   end
+
+  describe '#prettyprint_hosts' do
```

Comment:
Yeah....I feel like that's fine for the most part. I'm not too experienced in webmock unit tests checking stdout from puts but I feel like it should be ok? Is it brittle just because the moment someone changes how the prettyprint hosts method works it'll break? or because there's the potential for other things to appear in stdout while running unit tests?